src/SFML/Graphics/Font.cpp

```diff
@@ -757,7 +760,7 @@ bool Font::setCurrentSize(unsigned int characterSize) const
 
 
 ////////////////////////////////////////////////////////////
-Font::Page::Page(bool smooth)
+std::optional<Font::Page> Font::Page::make(bool smooth)
```

Comment:
Any reason we didn't use `create` as with most other factory functions?

Reply:
[Good point](https://github.com/SFML/SFML/pull/3103)